types/jasmine/index.d.ts

```diff
@@ -436,58 +483,66 @@ declare namespace jasmine {
 
     interface Env {
         addReporter(reporter: CustomReporter): void;
-
-        execute(): void;
-        describe(description: string, specDefinitions: () => void): Suite;
-        // ddescribe(description: string, specDefinitions: () => void): Suite; Not a part of jasmine. Angular team adds these
-        beforeEach(beforeEachFunction: ImplementationCallback, timeout?: number): void;
-        beforeAll(beforeAllFunction: ImplementationCallback, timeout?: number): void;
-        afterEach(afterEachFunction: ImplementationCallback, timeout?: number): void;
-        afterAll(afterAllFunction: ImplementationCallback, timeout?: number): void;
-        xdescribe(desc: string, specDefinitions: () => void): XSuite;
-        it(description: string, func: () => void): Spec;
+        allowRespy(allow: boolean): void;
```

Comment:
I do not see this in https://jasmine.github.io/api/3.7/Env

Reply:
Can you mark this as `@deprecated`? It seems like it's a preexisting type that should no longer exist.

Reply:
https://github.com/jasmine/jasmine/commit/2fc5182ddcdd67456b89c92c2eedbc2443c928c6#diff-46cdc2a30e1879f73f5a62a709840911600e79ff44702785f1fafb4234789c57R1919

This is not deprecated